renovate.json

```diff
@@ -2,5 +2,14 @@
   "extends": ["config:base"],
   "kubernetes": {
     "fileMatch": ["(^|/)[^/]*\\.yaml$"]
-  }
+  },
+  "prHourlyLimit": 0,
+  "packageRules": [
+    {
+      "packagePatterns": [
+        "^sourcegraph/"
+      ],
+      "groupName": "sourcegraph docker images"
```

Comment:
nit: I'd capitalize "Sourcegraph" and "Docker" because it's used int the PR title